bambi/tests/test_built_models.py

```diff
@@ -36,38 +36,25 @@ def crossed_data():
 
 def test_empty_model(crossed_data):
```

Comment:
This used to test that two different ways to create models were actually equivalent, but now it is just creating the same model in the same way twice.

Reply:
You are right, and I was aware of that before the PR. I just didn't know if it was good to delete all the tests with this pattern. I decided to leave them just to see if fitting the same model twice, gives the same result both times... is it too dumb? do you suggest deleting those tests?

Reply:
Remove them or just test that models build once.

Reply:
Second option sounds better